test_scripts/RC/ButtonPress/005_check_rpc_paramters_for_button_press.lua

```diff
@@ -0,0 +1,141 @@
+---------------------------------------------------------------------------------------------------
+-- RPC: ButtonPress
+-- Script: 005
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local commonRC = require('test_scripts/RC/commonRC')
+local runner = require('user_modules/script_runner')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+
+--[[ Local variables for tests ]]
+local climate_button_names = {
+    "AC_MAX",
+    "AC",
+    "RECIRCULATE",
+    "FAN_UP",
+    "FAN_DOWN",
+    "TEMP_UP",
+    "TEMP_DOWN",
+    "DEFROST_MAX",
+    "DEFROST",
+    "DEFROST_REAR",
+    "UPPER_VENT",
+    "LOWER_VENT"
+}
+
+local radio_button_names = {
+    "VOLUME_UP",
+    "VOLUME_DOWN",
+    "EJECT",
+    "SOURCE",
+    "SHUFFLE",
+    "REPEAT",
+}
+
+local climate_params = {
+    moduleType = "CLIMATE",
+    buttonName = "AC_MAX",
+    buttonPressMode = "SHORT" -- LONG, SHORT
+}
+
+local radio_params = {
+    moduleType = "RADIO",
+    buttonName = "VOLUME_UP",
+    buttonPressMode = "LONG"
+}
+
+commonRC.timeout = 1000
+
+--[[ Local Functions ]]
+local function reset_climate_params( ... )
+    return {moduleType = "CLIMATE", buttonName = "AC", buttonPressMode = "SHORT"}
+end
+
+local function reset_radio_params( ... )
+    return {moduleType = "RADIO", buttonName = "VOLUME_UP", buttonPressMode = "SHORT"}
+end
+
+local function SendButtonPressPositive(button_press_params, self)
+    local cid = self.mobileSession:SendRPC("ButtonPress", button_press_params)
+
+    EXPECT_HMICALL("Buttons.ButtonPress", {
+        appID = self.applications["Test Application"],
+        moduleType = button_press_params.moduleType,
+        buttonName = button_press_params.buttonName,
+        buttonPressMode = button_press_params.buttonPressMode
+    })
+    :Times(1)
+    :Do(function(_, data)
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+
+    EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+
+    commonTestCases:DelayedExp(commonRC.timeout)
```

Comment:
accepted, corrected in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1574/commits/b4f76da576e577944521d50663b84a1780ffa2cb